build/archives/BUILD.gn

```diff
@@ -48,17 +48,45 @@ if (host_os == "linux") {
   }
 }
 
-zip_bundle("flutter_patched_sdk") {
-  output = "flutter_patched_sdk.zip"
-  deps = [ "//flutter/lib/snapshot:strong_platform" ]
-  files = [
-    {
-      source = "$root_out_dir/flutter_patched_sdk/vm_outline_strong.dill"
-      destination = "vm_outline_strong.dill"
-    },
-    {
-      source = "$root_out_dir/flutter_patched_sdk/platform_strong.dill"
-      destination = "platform_strong.dill"
-    },
-  ]
+# This is the flutter_sdk from debug configuration.
```

Comment:
Consider adding a comment about this being used for profile mode as well.

Reply:
done!